### DISCUSSION TITLE
Is it possible to disable daisyui styles to a particular component (div and its children) ?

### DISCUSSION THREAD

**Discussion by krishna-santosh** [saadeghi/daisyui#2113] [Q&A] [answered]
Default styling provided by daisyui is affecting an element that I found on uiverse.io, Is it possible to disable daisyui styling to that particular element (div and its children)? setting `styled` to false in tailwind config removes all the styling.. but I only want that particular element to not be affected by daisyui

**Comment by saadeghi** [comment, marked as answer]
You can use the `prefix` config to add a prefix to all daisyUI class name.  

Or you can use a PostCSS plugin like this to remove a specific class name: https://www.npmjs.com/package/postcss-remove-classes
